brainscore/benchmarks/search.py

```diff
@@ -85,3 +85,84 @@ def ceiling(self):
 
     def get_raw_data(self):
         return self.cumm_perf, self.saccades
+
+KlabZhang2018search_assembly = ['klab.Zhang2018search_waldo', 'klab.Zhang2018search_naturaldesign']
+KlabZhang2018search_assembly_ceil = [1, 1]

```

Comment:
is this the actual computed ceiling of the data? i.e. humans predict each other at 100%?

Reply:
seems you are looking at the outdated version. please check the latest one. i tagged you there.